model/src/main/proto/exploration.proto

```diff
@@ -331,3 +334,21 @@ message HelpIndex {
     bool everything_revealed = 3;
   }
 }
+
+// Different states in which exploration checkpoint can exist.
+enum CheckpointState {
+
```

Comment:
Nit: remove extra newline at the beginning of the message for consistency with other enums/messages.

Reply:
done

Reply:
Marking as resolved as already done